packages/react-native-codegen/src/parsers/typescript/components/__test_fixtures__/fixtures.js

```diff
@@ -857,12 +857,12 @@ export interface ModuleProps extends ViewProps {
     'paperDirectEventDefinedInlineNullWithPaperName'
   > | null | undefined;
 
-  onBubblingEventDefinedInlineNull: BubblingEventHandler<null>;
-  onBubblingEventDefinedInlineNullOptionalKey?: BubblingEventHandler<null>;
-  onBubblingEventDefinedInlineNullOptionalValue: BubblingEventHandler<null> | null | undefined;
-  onBubblingEventDefinedInlineNullOptionalBoth?: BubblingEventHandler<null> | null | undefined;
+  onBubblingEventDefinedInlineNull: BubblingEventHandler<undefined>;
+  onBubblingEventDefinedInlineNullOptionalKey?: BubblingEventHandler<undefined>;
+  onBubblingEventDefinedInlineNullOptionalValue: BubblingEventHandler<undefined> | null | undefined;
+  onBubblingEventDefinedInlineNullOptionalBoth?: BubblingEventHandler<undefined> | null | undefined;
   onBubblingEventDefinedInlineNullWithPaperName?: BubblingEventHandler<
```

Comment:
I'm not a super expert, but what's the difference between `undefined` and `null` in this context? I had the impression that `null` is more idiomatic for TypeScript than undefined.

Also, if we use `undefined` we should also rename the events: from `onBubblingEventDefinedInlineNull` to `onBubblingEventDefinedInlineUndefined`.

Plus, we should probably keep both, events with `undefined` and events with `null`.

Reply:
`DirectEventHandler` cases above are still `null`. I think there is no reason to only support `null` if both `null` and `undefined` are equivalent in other places.